magento/pwa-studio
# [#2141] Fix wrong PriceSummaryQuery export

## Description

It fixes wrong export (https://github.com/magento/pwa-studio/issues/2141) 

## Related Issue





closes #2141.

## Acceptance 



### Verification Stakeholders
@sirugh 

### Verification Steps


1. Clone this PR
2. Just run yarn watch:venia
3. You won't see warning in console
